pages/api/paybutton/download/transactions/[paybuttonId].ts

```diff
@@ -0,0 +1,136 @@
+import Cors from 'cors'
+import moment from 'moment'
+import {
+  CAD_QUOTE_ID,
+  DEFAULT_QUOTE_SLUG,
+  PRICE_API_DATE_FORMAT,
+  USD_QUOTE_ID,
+  SUPPORTED_QUOTES,
+  RESPONSE_MESSAGES,
+  DEFAULT_PAYBUTTON_TRANSACTIONS_FILE_DELIMITER,
+  PAYBUTTON_TRANSACTIONS_FILE_HEADERS
+} from 'constants/index'
+import { TransactionWithAddressAndPrices, fetchTransactionsByPaybuttonId } from 'services/transactionService'
+import { PaybuttonWithAddresses, fetchPaybuttonById } from 'services/paybuttonService'
+import { streamToCSV } from 'utils/files'
+import { setSession } from 'utils/setSession'
+import { NextApiResponse } from 'next'
+import { TransactionFileData, FormattedTransactionFileData } from 'types'
+type SupportedQuotesType = typeof SUPPORTED_QUOTES[number]
+
+const cors = Cors({ methods: ['GET', 'HEAD'] })
+
+const getPrice = (transaction: TransactionWithAddressAndPrices, currency?: SupportedQuotesType): number => {
```

Comment:
- Better name: get transaction cost / value; since the price is already there in the transaction and what we want is to multiply it for the crypto amount.
- This could be in services/transactionService. Getting how much a tx in was valued in total looks generic enough to be there IMO

Reply:
These four first lines in the function could just be 
`const stringCurrency = currency === '' || currency === undefined ? DEFAULT_QUOTE_SLUG : currency`

and then below, instead of 
`return (!isCurrencyEmptyOrUndefined ? result[stringCurrency] : result.usd)`

just
`return result[stringCurrency]`


...unless I am missing something?